components/alert/alert-toast.js

```diff
@@ -331,18 +355,29 @@ class AlertToast extends LitElement {
 	}
 
 	_onFocus() {
+		ALERT_HAS_FOCUS = true;
```

Comment:
Presumably when you're tabbing from one alert to the next, the blur for the previous one fires first (setting `ALERT_HAS_FOCUS` to `false`) and _then_ the focus for the next one fires (setting it back to `true`)?

Reply:
Good question - tested this out (Chrome, Safari, Firefox) and it does seem to be the case that consistently blur will happen first and set `ALERT_HAS_FOCUS` to `false` and then focus will happen on the next alert